lib/msal-electron-proof-of-concept/src/appConfig/applicationConfiguration.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * @type AuthOptions: Used to configure the auth options in the ApplicationConfiguration object.
+ * - clientId       - Client ID of your app registered with our Application registration portal : https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredAppsPreview in Microsoft Identity Platform
+ * - authority      - You can configure a specific authority, defaults to " " or "https://login.microsoftonline.com/common"
+ * - redirectUri    - The redirect URI of the application, this should be same as the value in the application registration portal. Defaults to `msal://auth`.
```

Comment:
> s [](start = 73, length = 2)

nit: add "the" between "should be" and "same"

also, if they want to use the default redirectUri, will they still need to register that in the portal? if so, that is not clear.

Reply:
Done